tests/routes/master/workers.test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var mockery = require('mockery');
+var rewire = require('rewire');
+var path = require('path');
+var assert = require('chai').assert;
+
+var __ROOTDIR = path.join(__dirname, '..', '..', '..');
```

Comment:
@nickwph that seems like a good idea; it would simplify all the paths. maybe create an issue for refactoring?